### PR TITLE
Image help

### PR DESCRIPTION
**Pull request by runleonarun**
## Description & motivation

This PR attempts to adjust image size using CSS instead of divs in code.

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

**Comment by runleonarun** [comment]
@JKarlavige Does this PR adjust image sizes in the correct way? I want to use the advice of @chuise to dictate image size using CSS instead of in the code.

I might add this to `current` if it's OK

**Comment by runleonarun** [comment]
cc @nghi-ly so you can see how to describe images

File: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-generating-documentation.md [hunk 1]
```diff
@@ -6,7 +6,12 @@ description: "Automatically generate project documentation as you run jobs."
 
 ## Enabling documentation
 
-Documentation can be enabled for a job in dbt Cloud by checking the "Generate docs?" checkbox on the Job Settings page, as shown below. Once this setting is enabled, subsequent runs of the job will automatically include a step to generate documentation.
+You can enable Documentation for a job in dbt Cloud when you edit your job settings or create a new job. Once you enable this setting, subsequent runs of the job will automatically include a step to generate documentation.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-generating-documentation.md]: Are we enabling documentation for a project? Is only one job designated to generate docs?

**Review by JKarlavige — APPROVED** [review]
This looks good to me, the css edit works well. One note, the Lightbox images could still be added without setting the width in the code, but it originally defaulted to 600px width.

So this is ultimately bumping all Lightbox images on the site from `600px` width to `400px` width. As long as that is intended this is good to go 👍

**Review by jeremyhutt11 — APPROVED** [review]
Looks good!

**Comment by runleonarun** [comment]
Note that I am temporarily adjusting the CODEOWNERS to help reduce notifications @dataders and team don't need to see.

**Pull request merged**